### PR TITLE
Big Blanket of Cyrogaia Tweaks

### PR DESCRIPTION
**Pull request by quackjack**
This PR is the result of a poll I made almost a week ago about what the people would want to see mapped 
https://strawpoll.com/LVyK8AqNkn0/results
poll2h

Added experimental new "slum" dorms, located near actual dorms in the maints
I wanted to experiment with the subtle theme of the steamy hotspring from which the facility was built upon, so in combination, i've implemented those hotspring pools in select spots in maint
The aim with the slum dorms is to be incomplete, trashy and missing many standard things normal dorms have like actual beds; being a hassle to fix up into something nice but not making it impossible.
slumd

Something else from the poll was surprising, was a want for more merchandise stands to buy and sell things from in RP on the first floor
Despite it's popularity on the poll, I wanted to go a little light on acting on it initially to understand how much want there was for this; which wasn't too much considering the manifest of active players
Surprisingly there are very few Assistant/Entreprenur players, from what I saw there was a very very few amount of mercantile going around, so for this I turned the windowed maint spot on the hotspring walkway by dorms into the first of the shop stands for now.
springvein

Another area I wanted to tweak but for the second, main floor was the Gateway and connected lockerroom
Usually what I saw from other vorecode servers, was that of a more tucked and compact gateway design that had great markings but, with an amount of space that made it feel like a super distant teleportation device was rather unimportant
For the locker room, I didn't change much except the center's layout, having some fun with markings and shadows; although there's some more I could do if I allotted more space for this room
gatew

Among these changes, I also fixed up some things all around the 1st floor/stuff from my other changes to spruce things, and some secrets.
mats

### CODE REVIEW

**Comment by Repede** [comment]
Yo, really like the changes, honestly all that will just help add a bit of a breath of fresh air to the place for sure.

I didn't look at the diffs but I saw you're adding armed beartraps in the maints, I forget if they deal damage but it could be a bit annoying for maint dwellers, though I'm not against it if it's harmless.

**Comment by quackjack** [comment]
> Yo, really like the changes, honestly all that will just help add a bit of a breath of fresh air to the place for sure.
> 
> I didn't look at the diffs but I saw you're adding armed beartraps in the maints, I forget if they deal damage but it could be a bit annoying for maint dwellers, though I'm not against it if it's harmless.

I added a new 66/33 trap spawner for armed beartraps/mousetraps, but only about 2 instances of it on the map. It was more for  having a mapping helper for traps in the future/armed beartraps

**Comment by Repede** [comment]
> > Yo, really like the changes, honestly all that will just help add a bit of a breath of fresh air to the place for sure.
> > I didn't look at the diffs but I saw you're adding armed beartraps in the maints, I forget if they deal damage but it could be a bit annoying for maint dwellers, though I'm not against it if it's harmless.
> 
> I added a new 66/33 trap spawner for armed beartraps/mousetraps, but only about 2 instances of it on the map. It was more for having a mapping helper for traps in the future/armed beartraps

Do you know if the bear trap deal damage or just stun is what I meant, it's cool either way I think though. More out of curiosity.

**Comment by quackjack** [comment]
> > > Yo, really like the changes, honestly all that will just help add a bit of a breath of fresh air to the place for sure.
> > > I didn't look at the diffs but I saw you're adding armed beartraps in the maints, I forget if they deal damage but it could be a bit annoying for maint dwellers, though I'm not against it if it's harmless.
> > 
> > 
> > I added a new 66/33 trap spawner for armed beartraps/mousetraps, but only about 2 instances of it on the map. It was more for having a mapping helper for traps in the future/armed beartraps
> 
> Do you know if the bear trap deal damage or just stun is what I meant, it's cool either way I think though. More out of curiosity.

Bear traps stun for 2.5 seconds, and do 30 brute

File: code/game/objects/items/weapons/traps.dm [hunk 1]
```diff
@@ -394,3 +394,7 @@
 /obj/item/weapon/material/barbedwire/plastic
 	name = "snare wire"
 	default_material = MAT_PLASTIC
+
+/obj/item/weapon/beartrap/armed
```
Comment (izac112) [inline code/game/objects/items/weapons/traps.dm:398]:
```suggestion
// YW ADDITION
/obj/item/weapon/beartrap/armed
```

**Pull request merged**